src/components/AppInstanceLabel.js

```diff
@@ -54,7 +54,7 @@ const Main = styled.div`
   )}
 `
 
-const StyledBadge = styled(Badge.App)`
+const StyledTag = styled(Tag).attrs({ mode: 'app' })`
```

Comment:
Does this custom styling still apply with `Tag`'s new styles?

Reply:
I don't think mode `app` is done but when it is, yes